static-build/qemu/Makefile

```diff
@@ -0,0 +1,10 @@
+MK_DIR :=$(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
+CONFIG_DIR := $(MK_DIR)/../../scripts/
+QEMU_REPO := https://github.com/qemu/qemu.git
+QEMU_VERSION := stable-2.11
```

Comment:
I'd add this to `versions.txt`.

Aside: I thought we used to supplement the output of `qemu-lite-system-x86_64 --version` with `cc` or something to show that it was a custom build? I think we should re-add that but also add `-static-` to the version as that would ensure `kata-runtime kata-env` would show all the details (and avoid the user having to run `file /path/to/qemu`).

Reply:
@jodh-intel  do you know what is the option I should use to add `static` to version string ?

Reply:
I think I fount it , PKGVERSION variable.

Reply:
Yep - that sounds familiar.